src/reference.yaml

```diff
@@ -7746,6 +8025,14 @@ paths:
               category: reference
               message: reference can not be longer than 255 character(s)
               params: ''
+            - code: charge_type_not_found
+              category: charge_type
+              message: Not available charge_type
+              params: ''
+            - code: invalid_charge_type
+              category: charge_type
+              message:  Invalid banking_charges supplied, must be in range [:ours, :shared]
```

Comment:
one thing here, please edit this message to be like that:
```Invalid banking_charges supplied, must be in range [ours, shared]```
just without dots
@v-osadchuk

Reply:
done, thanks @Max43